### ISSUE TITLE
ToolTip: When 'focus' and 'hover' events happen at the same time, tooltips are getting overlapped

### ISSUE THREAD

**Issue by rinnuagna** [primefaces/primereact#5727]
### Describe the bug
![image](https://github.com/primefaces/primereact/assets/43952574/e0b95d66-d6f1-4021-b01d-8a5bad752cc6)

When 'focus' and 'hover' events happen at the same time for an icon and if we hover on the other icon, the tooltips get overlapped making it hard to read. This happens when the icons are too close. 

### PrimeReact version

10.2.1

### React version

17.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Browser(s)

All browsers

**Comment by melloware** [comment]
Similar to #6008

**Comment by dungnguyenBKA** [comment]
Hello, is any process on this bug?

**Comment by melloware** [comment]
Nope unless you want to sponsor it financially @dungnguyenBKA

**Comment by Luko248** [comment]
I have an simular issue, if you use event="both", it doesn't work properly. It how tooltip on hover but when focus input, it do nothink. So it works same as event="hover" https://primereact.org/tooltip/#event I really need to fix this bug.

**Comment by rinnuagna** [comment]
yes this is still happening in your latest version as shown in the above image
